x/spec/keeper/grpc_query_show_all_chains.go

```diff
@@ -26,14 +26,26 @@ func (k Keeper) ShowAllChains(goCtx context.Context, req *types.QueryShowAllChai
 		chainName := spec.GetName()
 		chainId := spec.GetIndex()
 
+		expectedInterfaces := make(map[string]bool)
```

Comment:
why do we need the make here?

Reply:
Because the internal function expects a map and updates it

Reply:
but it receives an empty map and than you override the existing one' why not just create the map inside the method?